### PR TITLE
Gear VR controller script + example

### PR DESCRIPTION
**Pull request by servinlp**
A script for using the Gear VR controller with an example.

Assets are the model from A-Frame (with license) and a skybox (with license).

### CODE REVIEW

**Comment by Mugen87** [comment]
For testing: http://rawgit.com/servinlp/three.js/gear-vr-controller-with-example/examples/webvr_gear_vr.html

File: examples/js/vr/GearVRController.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+		}
+
+	}
+
+	this.setSkyBox = function ( skyBox ) {
+
+		if ( 'SamsungChangeSky' in window ) {
+
+			window.SamsungChangeSky( skyBox );
```
Comment (mrdoob) [inline examples/js/vr/GearVRController.js]: I'm not sure this should be a feature of the `GearVRController`?

Comment (servinlp) [reply examples/js/vr/GearVRController.js]: So remove it and leave it up for the developers?

Comment (mrdoob) [reply examples/js/vr/GearVRController.js]: I think so, yeah.
We also show be careful of not adding to many images to the repo. It's already pretty big.

Comment (mrdoob) [reply examples/js/vr/GearVRController.js]: Also... Is `SamsungSky` more performant than using `scene.background`?

Comment (servinlp) [reply examples/js/vr/GearVRController.js]: The SamsungSky function will change the skybox before you enter webvr. Not inside Three.js
See http://developer.samsung.com/internet/gearvr/web-guide for this

Comment (servinlp) [reply examples/js/vr/GearVRController.js]: As for the images i'll see if i can downsize them

Comment (mrdoob) [reply examples/js/vr/GearVRController.js]: Why not using one of the skyboxes textures already available?
https://github.com/mrdoob/three.js/tree/dev/examples/textures/cube

Comment (servinlp) [reply examples/js/vr/GearVRController.js]: Sorry wasn't aware of that. I'll edit it.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**